anomdetect/src/main/java/com/expedia/adaptivealerting/anomdetect/comp/legacy/LegacyDetectorFactory.java

```diff
@@ -46,30 +41,36 @@
 @Slf4j
 @Deprecated
```

Comment:
Would be nice to link to whatever is replacing this class or some sort of description.

Reply:
It doesn't exist yet, but I will.